app/server.R

```diff
@@ -131,14 +131,14 @@ server <- function(input, output, session) {
   HOSP_CURRENT <- resolveCurrentHospDay()
 
   PREV_AS_OF_DATA <- reactiveVal(NULL)
-  AS_OF_CHOICES <- reactiveVal(NULL)
+  AS_OF_CHOICES <- reactiveVal(resolveCurrentHospDay())
```

Comment:
It looks to me that the main goal of `AS_OF_CHOICES` being reactive is to be easy to update from child processes. It doesn't look like its reactive behavior will ever be triggered since as of choices will only change when the target variable changes, which causes the same replotting and recalculating that `AS_OF_CHOICES` would.

I think we're safe to make it into a global instead, but then we'll need to do out-of-environment assignment (`<<-`). Knowing that, do you have a preference one way or the other?

Reply:
hmmm I don't feel strongly about it -- a global vs some superfluous reactivity are probably equally annoying. that suggests, do whatever's easy :)

Reply:
Given that we should do a bigger refactor of the reactive behavior, I'm leaving this as-is and will create an issue.